contrib/win32/win32compat/ansiprsr.c

```diff
@@ -176,14 +178,10 @@ unsigned char* ParseBuffer(unsigned char* pszBuffer, unsigned char* pszBufferEnd
 			unsigned char * pszCurrent = pszBuffer+1;
 			unsigned char * pszNewCurrent = pszCurrent;
 
-			if (term_mode == TERM_ANSI)
+			if (term_mode == TERM_ANSI && bAnsiParsing)
```

Comment:
what is the client behavior if the TERM is not set to TERM_ANSI?

can we have an error message that we only support TERM_ANSI as of now because XTERM or others fail at some point.